swap_meet/decor.py

```diff
@@ -1,2 +1,7 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, category = "Decor", condition = 0, age = 0):
+        super().__init__(category, condition, age)
```

Comment:
Tiny style point here and elsewhere: when defining default arguments don't include spaces on either side of the equals sign. We would instead write it like:

```
def __init__(self, category="Decor", condition=0, age=0):
```